addons/my_app/views/assets.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="assets_frontend" inherit_id="website.assets_frontend" name="Assets Frontend">
```

Comment:
```suggestion
    <template id="assets_frontend" inherit_id="theme_a.assets_frontend" name="Assets Frontend">
```

To use the variables created in another module, you need to intherit the assets of that module and not from website.

Do this change and you will not have the Undefined variable error.

Reply:
Hello @jpp-odoo ,

so, I should inherit each inheritance of website.assets tempalte? I mean to inherit all theme_x.assets, and all website_x_module.assets in a different tag template if I want use them just for a variable?

Regards!